src/controller/DirectoryGraph.cpp

```diff
@@ -1,10 +1,13 @@
 #include "DirectoryGraph.h"
 #include <iostream>
 #include <algorithm>
+#include <string>
 
 using namespace std;
 
-DirectoryGraph::DirectoryGraph()
+DirectoryGraph::DirectoryGraph(std::set<std::string> &exludedDirectories, bool showHidden)
```

Comment:
I did change the constructor to load in the file too, however i don't see how it would be able to take in showHidden as that is provided as an argument.